conf/waives

```diff
@@ -0,0 +1,53 @@
+# TODO: unknown, but maybe expected as the rule description says that
+#   "The remediation for this rule uses firewall-cmd and nmcli tools.
+#    Therefore, it will only be executed if firewalld and NetworkManager
+#    services are running."
+# and since these are not running in an Anaconda %post environment,
+# maybe the remediation failing is expected .. ?
+# - see also https://github.com/ComplianceAsCode/content/issues/1639
```

Comment:
Very old issue, the rule has been reworked.
Use https://github.com/ComplianceAsCode/content/issues/9746

Reply:
Thanks, will change the comment. The curious part is that this still seems to be failing even with current upstream content from git.

Reply:
Well, the issue is closed because it is not a problem in content. The problem is in Anaconda environment and we can't affect that, so it will fail there most likely forever.